charts/jenkins/unittests/jenkins-controller-ingress-DRY-test.yaml

```diff
@@ -0,0 +1,46 @@
+suite: Controller Ingress - DRY Support
+release:
+  name: my-release
+  namespace: my-namespace
+templates:
+  - jenkins-controller-ingress.yaml
+  - jcasc-config.yaml
+tests:
+  - it: controller-ingress
+    template: jenkins-controller-ingress.yaml
+    set:
+      global.jenkinsHostname: "jenkins.example.com"
+      controller.ingress:
+        enabled: true
+        hostName: "{{ .Values.global.jenkinsHostname }}"
+        resourceRootUrl: "{{ .Values.global.jenkinsHostname }}"
+        tls:
+          - hosts:
+            - "{{ .Values.global.jenkinsHostname }}"
+    asserts:
+      - equal:
+          path: spec.rules[0].host
+          value: "jenkins.example.com"
+      - equal:
+          path: spec.tls[0].hosts[0]
+          value: "jenkins.example.com"
+#  - it: jcasc-config
+#    template: jcasc-config.yaml
+#    set:
+#      global.jenkinsHostname: "jenkins.example.com"
+#      controller.ingress:
+#        enabled: true
+#        hostName: "{{ .Values.global.jenkinsHostname }}"
+#        resourceRootUrl: "{{ .Values.global.jenkinsHostname }}"
+#        tls:
+#          - hosts:
+#            - "{{ .Values.global.jenkinsHostname }}"
+#    asserts:
+#      - exists:
+#          path: data.[jcasc-default-config.yaml]
+#      - isNotNullOrEmpty:
+#          path: data.[jcasc-default-config.yaml]
+#      - matchRegex:
+#          path: data.[jcasc-default-config.yaml]
+#          pattern: |
+#            url: https://jenkins.example.com
```

Comment:
I think we can remove this, doesn't give much value as-is given the framework doesn't support it, happy to approve and merge after
```suggestion
```

Reply:
> I think we can remove this, doesn't give much value as-is given the framework doesn't support it, happy to approve and merge after

@timja 
Removal of the commented test case done...